src/components/RadonOperator.vue

```diff
@@ -397,15 +414,45 @@ export default {
     :scriptId=1
     :operator="{
       selected:{
-        label: 'asFloat',
-        outputType: 'float',
+        label: 'map',
```

Comment:
I'd keep this example as simplest as possible due to we already have a example with subscript below